### PR TITLE
virtual_ship package

### PR DESCRIPTION
**Pull request by surgura**
- Packaged code in virtual_ship package
- Created pyproject.toml for project
- Made tiny changes to code that were still on Emma's computer and not on github

It's just a first iteration, we will package and organize the code more thoroughly later.

@ammedd I left open the project description in the toml file for you to create.

### CODE REVIEW

File: old_code.py
Comment (erikvansebille) [inline old_code.py:1]: Why is this `old_code.py` file needed? Is it the old `virtualship.py`? Can't we just get it from the git history?

File: .gitignore [hunk 1]
```diff
@@ -169,3 +169,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Auto generated by setuptools scm
+virtual_ship/_version_setup.py
```
Comment (erikvansebille) [inline .gitignore]: Add a hard return at tend of file
```suggestion
virtual_ship/_version_setup.py

```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = ["setuptools >= 61.0", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "virtual_ship"
+description = "Emma's work for the MSc student material. TODO Emma make a good description"
```
Comment (erikvansebille) [inline pyproject.toml:7]:
```suggestion
description = "Code for the Virtual Ship Classroom, where Marine Scientists can combine Copernicus Marine Data with an OceanParcels ship to go on a virtual expedition."
```

File: virtual_ship/costs.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+def costs(config, total_time):
+    '''Calculates cost of the virtual ship'''
```
Comment (erikvansebille) [inline virtual_ship/costs.py:2]:
```suggestion
    '''Calculates cost of the virtual ship (in US$)'''
```

**Review by erikvansebille — APPROVED** [review]
Good package structure, see a few suggestions and a question below

File: old_code.py
Comment (surgura) [reply old_code.py:1]: Need to discuss with Emma what code is relevant and what is not. Old code contains the code that is not used except by commented out lines. After this pr I will do a larger cleanup where we remove all that stuff. I just wanted to get the basics out there.

**Comment by surgura** [comment]
Emma will first push the final code she still had locally on her computer. Then I will rearrange this pr.

**Comment by ammedd** [comment]
Go ahead with the PR, I'll add things as issues

File: old_code.py [hunk 4]
```diff
@@ -0,0 +1,237 @@
+import json
```
Comment (ammedd) [inline old_code.py:1]: Not old!

**Review by ammedd — CHANGES_REQUESTED** [review]

**Review by ammedd — APPROVED** [review]

**Review by erikvansebille — APPROVED** [review]

**Pull request merged**